portfolio/src/main/webapp/index.html

```diff
@@ -22,11 +22,19 @@ <h1>Sena's Portfolio</h1>
       <p>This is a random feature that says "Hello world" in 4 languages:<p> 
       <button onclick="addRandomGreeting()">Hello</button>
       <p id="greetingContainer"></p>
-      <p>Why is that feature here? Je ne sais pas. 
-          <button onclick = "getJSONobject()">Get a quote from Thanos</button>
-      </p>
-      <ul id = "quoteSection"></ul>
+      <p>Why is that feature here? Je ne sais pas.</p>
     </div>
+    <p>Please leave below what you think about the site in the comments section:</p>
+    <form action = "/data", method = "POST">
+        <label for="name">Name:</label>
+        <input type = "text" id = "userName" name = "userName" value = "Your name here">
+        <br/>
+        <label for = "comment">Comment:</label>
+        <textarea type = "text" id = "comment" name = "comment">Your comment</textarea>
```

Comment:
for both the input and textarea, I'd recommend using the placeholder tag instead of having the default value be "your name here" or "Your comment" so that users don't have to delete the whole phrase.

Example --> <input type = "text" id = "userName" name = "userName" placeholder = "Your name here">

Reply:
Added feature to new pull request